### DISCUSSION TITLE
How can I add shortcuts to launch games directly?

### DISCUSSION THREAD

**Discussion by rmvz3** [Whisky-App/Whisky#138] [Q&A] [answered]
I am amazed with this new "feature" in my Mac and I want to thank you for make it A LOT easier to test it than the default way.
I was wondering how can I add an icon in whisky to launch apps directly. I mean, I've installed Steam and I got the Steam icon in my bottle but then I install a demo from Steam (Life of P) and I want to launch it directly without opening Steam first. Currently I have two Steam icons (refering the same steam.exe target) but I don't know if it is even possible to add the LoP.exe shortcut to the launcher

**Comment by andrewkeithly** [comment, marked as answer]
I can tell you how but as far as I can tell `.app` shortcuts for Steam and its games dont open properly. 

1. Go to your Bottle, for me this is "Windows 10" in sidebar.
2. Select "Installed Programs"
3. Locate the `.exe` file to the game you'd like to create a shortcut, right-click and select "config"
4. At bottom there should be an option to "Create Shortcut" at bottom.

**Comment by ryanbooker** [comment]
I see the same thing with other apps. The created shortcut doesn't actually work. It bounces in the dock briefly, then disappears.

**Comment by Salbawardi2** [comment]
I also tried what [andrewkeithly](https://github.com/andrewkeithly) suggested and had the same problem as [ryanbooker](https://github.com/ryanbooker). I'm wondering if anyone found a solution for this. Thanks!

**Comment by Salbawardi2** [comment]
Actually this worked for me: https://github.com/Whisky-App/Whisky/issues/725#issuecomment-1866765946

**Comment by contepaccio** [comment]
Hi, i've the same problem as @ryanbooker, the alias bounce for a while in the dock and then it disappears 🤷🏻‍♂️
I'm using a simple Win Portable App (a few files in a folder) not an installed game
